src/components/application_manager/include/application_manager/commands/hmi/rc_get_capabilities_request.h

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
@yang1070 
I see that number of files even in development branch have that part. I would propose to create separate PR to remove copyright from all over the project. Otherwise most files will be affected and history of changes won't look good.